go/vt/vtctl/reparentutil/replication.go

```diff
@@ -208,16 +216,26 @@ func StopReplicationAndBuildStatusMaps(
 		}
 	}
 
+	tabletAliasToWaitFor := ""
+	numErrosToWaitFor := 0
+	if tabletToWaitFor != nil {
+		tabletAliasToWaitFor = topoproto.TabletAliasString(tabletToWaitFor)
```

Comment:
Contortions because of the different types. Not much we can do about it I guess.

Reply:
Unfortunately, we can't. I tried sending a string into the function instead of the tabletAlias for the new parameter, but that wasn't helpful, since we would have to do the nil check outside in the `reparentShardLocked` function, which seemed an even worse design choice